README.md

```diff
@@ -5,14 +5,12 @@
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/2cf538781fdc4680a7103bcf96417a9a)](https://www.codacy.com/gh/e-valuation/EvaP/dashboard)
 [![codecov](https://codecov.io/gh/e-valuation/EvaP/branch/master/graph/badge.svg)](https://codecov.io/gh/e-valuation/EvaP)
 
-
-## What is EvaP?
+## What is EvaP
```

Comment:
it doesn't allow question marks in headings?

Reply:
[Yup](https://app.codacy.com/gh/e-valuation/EvaP/issues?bid=23052067&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6WyJNYXJrZG93biJdfSx7ImlkIjoiQ2F0ZWdvcnkiLCJ2YWx1ZXMiOltdfSx7ImlkIjoiTGV2ZWwiLCJ2YWx1ZXMiOltdfSx7ImlkIjoiUGF0dGVybiIsInZhbHVlcyI6WyI2NjI2Il19LHsiaWQiOiJBdXRob3IiLCJ2YWx1ZXMiOltdfV0=), the [no-heading-punctuation](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-heading-punctuation) rule is active by default. I don't actually care about the question mark here, I can see how someone could argue it to be a smell, but I'm also fine with just disabling the rule if you prefer that.

Reply:
Btw, same for the "no-multiple-empty-lines" rule. Personally, I don't really care if we put two empty lines before a new heading. With syntax highlighting, it shouldn't be necessary, but I don't think it does any damage.

Reply:
as discussed, two empty lines before the heading are nice but i really don't understand the problem with question marks in headings from a linguistic perspective. in this case, both solutions (removing the question mark, disabling the rule) are fine for me :)

Reply:
Removed the changes from README.md. Deactivated the two checks on codacy.